sqldelight-runtime/src/main/java/com/squareup/sqldelight/db/SqlResultSet.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.sqldelight.db
+
+interface SqlResultSet {
+  fun next(): Boolean
+  fun getString(index: Int): String
```

Comment:
I wonder if `get` as a prefix is worthwhile

Reply:
I'm not sure. I like it because we are "getting" something from the result set but maybe `stringAt(index = 1)` is a better api since it doesnt resemble java beans wrapping fields

Reply:
SGTM